### DISCUSSION TITLE
无法显示评论,评论区一直转圈 (WordPress)

### DISCUSSION THREAD

**Discussion by ThekingMX1998** [walinejs/waline#1237] [Q&A] [answered]
console 无报错 博客页面无法使用评论及登录功能

**出现问题的博客地址:**

[https://blog.greenfishpowered.cn](https://blog.greenfishpowered.cn)

**使用的WordPress主题**

[https://github.com/HelipengTony/tony](https://github.com/HelipengTony/tony)

**服务端**部署在 [Vercel](https://vercel.com) 

**数据库**使用 [LeanCloud](https://leancloud.app)

**具体情况如图:**

![image](https://user-images.githubusercontent.com/79040577/181788919-bb784f18-b8a5-408b-b516-c055ba790607.png)

**Comment by lizheming** [comment, marked as answer]
这个主题 `<main>` 区域都是使用 Vue 进行的渲染，会和 Waline Client 内部的 Vue 渲染有冲突。解决办法是不要把 Waline 渲染的区域放在 `<main>` 内部即可。

**Reply by ThekingMX1998** [reply]
> 这个主题 `<main>` 区域都是使用 Vue 进行的渲染，会和 Waline Client 内部的 Vue 渲染有冲突。解决办法是不要把 Waline 渲染的区域放在 `<main>` 内部即可。

非常感谢 成功解决了

**Comment by Mister-Hope** [comment]
或者你就魔改主题，让主题以组件的形式调用waline
